test/common.js

```diff
@@ -12,4 +12,3 @@ var portCounter = 9042
 global.nextPort = function() {
   return ++portCounter
 }
-
```

Comment:
This empty new line fix is not related to the issue, therefore it should go into a separate pull-request and reverted in this one.